src/test/resources/mockdata/assignments/assignments.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
This file could be stored in a folder named `acquisitionsUnitAssignments` more specific to actual API

Reply:
Renamed